### DISCUSSION TITLE
Browser-based frontend to emacs possible now?

### DISCUSSION THREAD

**Discussion by gtusr** [emacs-ng/emacs-ng#492] [General]
Chris Done once proposed creating a new front-end for emacs by leveraging CodeMirror:

[Idea: A web frontend for Emacs via CodeMirror](https://gist.github.com/chrisdone/f874ac1c872c004d6ce9e3cb20cc59e0)

Can emacs-ng help bring us any closer to realizing that dream?

**Comment by declantsien** [comment]
gtusr ***@***.***> writes:

> Chris Done once proposed creating a new front-end for emacs by leveraging CodeMirror:
>
> [Idea: A web frontend for Emacs via CodeMirror](https://gist.github.com/chrisdone/f874ac1c872c004d6ce9e3cb20cc59e0)
>
> Can emacs-ng help bring us any closer to realizing that dream?  
>
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/emacs-ng/emacs-ng/issues/491
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

Hey, gtusr

Have you tried PGTK in `broadway`?
I just did a little test. It seems pretty solid.
https://lists.gnu.org/archive/html/emacs-devel/2020-12/msg01442.html
[[https://docs.gtk.org/gtk3/broadway.html][Gtk – 3.0: Using GTK with Broadway]]

Regards,
Declan
